### DISCUSSION TITLE
The cities in my surrounding area are no longer showing on the Meeting Guide

### DISCUSSION THREAD

**Discussion by FresnoAAWebChair** [code4recovery/12-step-meeting-list#1289] [General]
The cities in my surrounding area are no longer showing on the Meeting Guide - The only city showing up is the local city "Fresno, Ca," - I check my meeting schedule on the website https://www.fresnoaa.org/meetings/ and they are there on the schedule - I am confuse if this is a issue on The Meeting Guide side or what? Please help - Thank you

**Comment by tim-rohrer** [comment]
Yes, if the meeting data is showing on your site, but not in Meeting Guide, then you will need to contact them to research why.

You didn't say, and I'm just curious, if the "surrounding areas" are being reported by district or similar websites?

**Reply by FresnoAAWebChair** [reply]
Who do I contact? As I am the Web Chair for Intergroup - It was reported to me when Central Office call me and said that one of the members of the fellowships in a city next to ours called and said their meetings wasnt there - when I check I notice none of the cities are showing except Fresno

**Reply by tim-rohrer** [reply]
[This](https://meetingguide.helpdocs.io/) is a good place to start. They have a contact button.

**Reply by FresnoAAWebChair** [reply]
Thanxs Tim - I did send them a email thru your link - hopefully it is something they can fix - Thank you again
